internal/pipeline/aggregate.go

```diff
@@ -328,18 +335,23 @@ func (xfagg *aggregator) notifyAfterUpload(filename string, file *ach.File, agen
 }
 
 func (xfagg *aggregator) notifyAboutHoliday(day *schedule.Day) {
+	var msg string
 	logger := xfagg.logger.With(log.Fields{
 		"shard": log.String(xfagg.shard.Name),
 	})
 
 	if !day.FirstWindow {
-		logger.Info().Log("skipping holiday notification")
+		msg = "skipping holiday notification"
+		xfagg.alertOnWarning(errors.New(msg))
+		logger.Info().Logf(msg)
 		return
 	}
 
 	uploadAgent := xfagg.uploadAgents.Find(xfagg.shard.UploadAgent)
 	if uploadAgent == nil {
-		logger.Warn().Logf("skipping holiday log for %v", day.Time.Format("2006-01-02"))
+		msg = fmt.Sprintf("skipping holiday log for %v", day.Time.Format("2006-01-02"))
+		xfagg.alertOnWarning(errors.New(msg))
```

Comment:
The holiday notification is optional, so not sending it isn't a warning.

Reply:
Oops, that was my mistake